### ISSUE TITLE
Provide new schema for convert package

### ISSUE THREAD

**Issue by LinguList** [lingpy/lingpy#45]
Currently, the convert package is handled in a rather inconsistent way. The file descriptions do not give us direct hints as to whether a module converts to the target (name of the module), or uses the target as input. Here, we need a restructuring of all modules that provides more consistency.

Especially, we should distinguish strictly between
- cases where stuff is _written_ to file (**write.py** ?),
- cases where _strings_ are produced (**string.py** ?),
- cases where _python_ data is produced (**internal.py** ?),
- cases where plots are produces (already distinguished with the plot-module),
- cases where functions are _open_ and applicable for users from outside (such as matrix2tree, where we can calculate a tree from a distance matrix), and those functions that are _privat_ (specific "ops" for wordlist, etc.).

Basically, we could rename the convert-modules as:
- file (or tofile, or write)
- string (or tostring)
- private (won't be loaded when calling `python from lingpy import *`)
- plot
- internal (or tointernal ???)

Ideas are welcome...
